README.md

```diff
@@ -1 +1,10 @@
 # testcafe-hammerhead
+
+Travic CI
+[![Build Status](https://travis-ci.org/churkin/testcafe-hammerhead.svg)](https://travis-ci.org/churkin/testcafe-hammerhead)
```

Comment:
We should change the url of Travis build status image to the upstream repo. Replace `churkin` to `superroma`